core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -59,7 +59,7 @@ enum CredentialAttrType
         private const SecAccessible _defaultAccessiblityPolicy = SecAccessible.AfterFirstUnlockThisDeviceOnly;
 
         private readonly string DefaultKeychainGroup = "com.microsoft.adalcache";
-        private readonly string TeamIdKey = "teamIDHint";
+        private readonly string TeamIdKey = ".NetTeamIDHint";
```

Comment:
1. Is this a breaking change for iOS cache wise?
2. Use const instead of readonly. 
3. Could you add a comment in the code on what this constant controls ?

Reply:
I'm still not sure why we really need to do this. I get that it may be failing on the test device you are using, however I assume it's possible to reset the device or remove the record from the device?

---
In reply to: [225133657](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1324#discussion_r225133657) [](ancestors = 225133657)

Reply:
1. no. Not a breaking change because we will write a new entry if the old one is not found. The reason this broke earlier is because we have to use SecAccessible.Always for TeamId.
2. Yes. Fixed
3. Yes. 

---
In reply to: [225223957](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1324#discussion_r225223957) [](ancestors = 225223957,225133657)

Reply:
@henrikm the default value is `AfterFirstUnlockThisDeviceOnly`, which is set when you don't specify anything (which is what we were doing previously). Now that we want to set the accessibility to something less restrictive (only for TeamId), we need to change the Team identifier. 

---
In reply to: [225223957](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1324#discussion_r225223957) [](ancestors = 225223957,225133657)